Source/WebInspectorUI/UserInterface/Views/ColorPicker.js

```diff
@@ -90,6 +90,18 @@ WI.ColorPicker = class ColorPicker extends WI.Object
         this._color = WI.Color.fromString("white");
 
         this._dontUpdateColor = false;
+        this._nodeStyles = nodeStyles || null;
+        if (this._nodeStyles && this._nodeStyles.length != 0) {
+            this._colorSwatchDisplay = new WI.ColorSwatchDisplay(this._nodeStyles);
+            this._colorSwatchDisplay.delegate = this;
+            this._element.appendChild(this._colorSwatchDisplay.element);
+        }
+    }
+
+    swatchSelected(variableName){
+        let resolvedValue = this._nodeStyles.computedStyle.resolveVariableValue(variableName);
```

Comment:
should be const instead of let

Reply:
FYI @rr-codes this is actually fine as it matches the pattern used in Web Inspector where `const` is only used for things that are unchanging between Web Inspector sessions (i.e. actual true constants)